### PR TITLE
feat(graphql): add CREDS_URL env variable

### PR DESCRIPTION
**Pull request by JeanBarriere**

### CODE REVIEW

File: templates/graphql/graphql.deployment.yaml [hunk 1]
```diff
@@ -32,3 +32,5 @@ spec:
                   key: public
             - name: WHITELIST_DOMAINS_REGEXP
               value: {{ .Values.graphql.corsWhitelistRegexp }}
+            - name: CREDS_URL
```
Comment (williammartin) [inline templates/graphql/graphql.deployment.yaml:35]: Can we make this consistent with the runtime env var? Either way is fine, but lets pick one

File: values.yaml [hunk 2]
```diff
@@ -68,6 +68,10 @@ creds:
     client_id:
     client_secret:
 
+geh:
+  image: storyscript/geh
+  slack_signing_key:
```
Comment (williammartin) [inline values.yaml:73]: And this should be `slack_signing_key`

File: templates/geh/geh.secrets.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .Release.Name }}-geh"
+type: Opaque
+data:
+  slack_signing_key: {{ required "A .Values.geh.slack_signing_key is required" .Values.geh.slack_signing_key | b64enc }}
```
Comment (williammartin) [inline templates/geh/geh.secrets.yml:7]: `slack_signing_secret`

File: templates/geh/geh.deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+          - name: http
+            containerPort: 3000
+          env:
+            - name: RUNTIME_URL
+              value: {{ printf "http://%s-runtime.%s.svc.cluster.local:9001" .Release.Name .Release.Namespace }}
+            - name: SLACK_SIGNING_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Release.Name }}-geh"
+                  key: slack_signing_key
```
Comment (williammartin) [inline templates/geh/geh.deployment.yaml:29]: `slack_signing_secret`

File: templates/geh/geh.deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        - name: geh
+          image: "{{ .Values.geh.image }}"
+          imagePullPolicy: Always
+          ports:
+          - name: http
+            containerPort: 3000
+          env:
+            - name: RUNTIME_URL
+              value: {{ printf "http://%s-runtime.%s.svc.cluster.local:9001" .Release.Name .Release.Namespace }}
+            - name: SLACK_SIGNING_KEY
```
Comment (williammartin) [inline templates/geh/geh.deployment.yaml:25]: SLACK_SIGNING_CREDENTIALS

File: templates/graphql/graphql.deployment.yaml (hunk 1 above)
Comment (JeanBarriere) [reply templates/graphql/graphql.deployment.yaml:35]: DONE

**Pull request merged**